fw/ringbuffer.c

```diff
@@ -0,0 +1,248 @@
+/**
```

Comment:
Technically this is the second lock-free RB in Tempesta FW and it's not generic. I'd rename the file to something pointing out that this is a kernel/user-space transport, not just an abstract data RB